vms/avm/states/state_test.go

```diff
@@ -99,15 +99,15 @@ func TestState(t *testing.T) {
 
 	db := memdb.New()
 	vdb := versiondb.New(db)
-	s, err := New(vdb, parser, prometheus.NewRegistry())
+	s, err := New(vdb, parser, prometheus.NewRegistry(), false)
```

Comment:
I like to do `, false /*trackChecksum*/)` instead of leaving naked args but up to you

Reply:
if we do that I'd prefer consts like `ChecksumEnabled` or `ChecksumDisabled`